insomniac.py

```diff
@@ -169,6 +182,58 @@ def on_likes_limit_reached():
                 raise e
 
 
+def _job_handle_hashtags(device, hashtags, likes_count, follow_percentage, storage, profile_filter, on_interaction):
+    class State:
+        def __init__(self):
+            pass
+
+        is_job_completed = False
+
+    state = State()
+    session_state = sessions[-1]
+
+    def on_likes_limit_reached():
+        state.is_job_completed = True
+
+    on_interaction = partial(on_interaction, on_likes_limit_reached=on_likes_limit_reached)
+
+    for hashtag in hashtags:
+        """is_myself = blogger == session_state.my_username
+        print_timeless("")
+        print(COLOR_BOLD + "Handle #" + hashtag + (is_myself and " (it\'s you)" or "") + COLOR_ENDC)"""
+        completed = False
+        #on_interaction = partial(on_interaction, blogger = hashtag)
```

Comment:
Could you remove commented lines pls? Just to keep clean

Reply:
Done